src/App.tsx

```diff
@@ -3,11 +3,11 @@ import './App.css';
 import { RenderRoutes } from './navigation/Router';
 import ROUTER_CONFIG from './navigation/router-config';
 
-const App: React.FC  =() => {
+const App: React.FC = () => {
   return (
     <Suspense fallback={<div>Загрузка...</div>}>
-      <div className='App'>
-        <header className='App-header' />
+      <div className="App">
```

Comment:
we use single quotoes, afaik.. 
why is it turned to double?

Reply:
we have a conflict with that rule, everyone who use `CTRL+S` in App.tsx or other components with JSX got double qoutes instead of single one's.